test/uk/gov/hmrc/tai/viewModels/TaxAccountSummaryViewModelSpec.scala

```diff
@@ -41,64 +41,66 @@ class TaxAccountSummaryViewModelSpec extends PlaySpec with FakeTaiPlayApplicatio
       "has header relating to current tax year" in {
         val expectedHeader = Messages("tai.incomeTaxSummary.heading.part1") + " " + currentTaxYearRange
 
-        val sut = TaxAccountSummaryViewModel(emptyTaxCodeIncomes, emptyEmployments, taxAccountSummary, ThreeWeeks, nonTaxCodeIncome)
+
+        val sut = TaxAccountSummaryViewModel(taxAccountSummary, ThreeWeeks, nonTaxCodeIncome, Seq(), Seq(), Seq(), Seq())
         sut.header mustBe expectedHeader
```

Comment:
Notice how we pass in lots of empties/we don't cares here, it suggest there's an object inside of TaxAccountSummaryViewModel that we don't care about and can be extracted out

Reply:
converted to case class for incomesources and used throughout